src/daemon/bitcoind/interface.rs

```diff
@@ -33,7 +33,7 @@ const RPC_SOCKET_TIMEOUT: u64 = 180;
 // Labels used to tag utxos in the watchonly wallet
 const DEPOSIT_UTXOS_LABEL: &str = "revault-deposit";
 const UNVAULT_UTXOS_LABEL: &str = "revault-unvault";
-const CPFP_UTXOS_LABEL: &str = "revault-cpfp";
+pub const CPFP_UTXOS_LABEL: &str = "revault-cpfp";
```

Comment:
meh. Didn't read the entire commit yet but it smells like a layer violation

Reply:
Made it a `pub(super) const`, slightly better :D I need this label in the poller to filter the cpfps in listunspent.

Or maybe i should have some method here to filter listunspent?